.github/workflows/ci.yml

```
@@ -0,0 +1,263 @@
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          name: release-package
+          path: dist/
+
+      - name: Upload release assets
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.10)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)